forge-game/src/main/java/forge/game/GameActionUtil.java

```diff
@@ -378,6 +352,72 @@ public static final List<SpellAbility> getAlternativeCosts(final SpellAbility sa
         }
         return alternatives;
     }
+    
+    /**
+     * Checks a card to see if it qualifies to receive keywords from static
+     * abilities that grant alternative casting costs to spells on the stack, for
+     * example, Toolbox's ability that gives blitz to creature spells with mana
+     * value 4 or greater, and Hunting Velociraptor's ability that gives prowl to
+     * Dinosaur spells.
+     * 
+     * @param source The card to check eligibility for.
+     * @return Zero or more spell abilities, each representing an alternative
+     *         casting cost.
+     */
+    public static final FCollectionView<SpellAbility> getAlternativeCostsGrantedByStaticAbilities(Card source) {
+        final FCollection<SpellAbility> alternatives = new FCollection<SpellAbility>();
+        final Game game = source.getGame();
+
+        if (!game.getAction().hasStaticAbilityAffectingZone(ZoneType.Stack, StaticAbilityLayer.ABILITIES)) {
+            return alternatives;
+        }
```

Comment:
The previous logic checked if the `source` is a creature, but I removed that here, because the restriction is already checked when applying the static ability in `checkStaticAbilities`, below.

This will allow static abilities that grant alternative costs to non-creature spells to function, also.